src/util.h

```diff
@@ -216,6 +216,21 @@ static SECP256K1_INLINE void memczero(void *s, size_t len, int flag) {
     }
 }
 
+/** Semantics like memcmp. Variable-time.
+   We use this to avoid possible compiler bugs with memcmp, e.g., a bug in certain versions of GCC 9 and 10. */
+int secp256k1_memcmp_var(const void *s1, const void *s2, size_t n) {
```

Comment:
Make `static` please.

Reply:
Are we required to follow the `const void *` signature?  Should we make it `const char *`?

Reply:
It seems that would need more code changes, as callers in some places expect to be able to pass a pointer to other objects.

Reply:
Oh I see.  This `void` thing is really subversive.